### PR TITLE
Speed up reload using file system watcher instead of checking every second

### PR DESCRIPTION
**Pull request by uxsoft**
This approach utilizes a file system watcher instead of computing a hash every second to see if the file changed. 

Should be on average 0.5s faster.

### CODE REVIEW

**Comment by uxsoft** [comment]
@worldbeater @AngelMunoz @MuminjonGuru @jl0pd

File: Live.Avalonia/LiveFileWatcher.cs [hunk 1]
```diff
@@ -2,55 +2,53 @@
...
     {
-        private readonly Subject<string> _fileChanged = new Subject<string>();
-        private readonly Action<string> _logger;
-        
-        private IDisposable _timerSubscription;
-        private string _latestSignature;
+        readonly Subject<string> _fileChanged = new Subject<string>();
+        readonly Action<string> _logger;
+
+        FileSystemWatcher watcher = new FileSystemWatcher()
```
Comment (worldbeater) [inline Live.Avalonia/LiveFileWatcher.cs:14]: Probably worth marking this field as `readonly`, as we aren't reassigning the watcher anywhere.

**Review by worldbeater — APPROVED** [review]
LGTM and thanks for doing this!

**Review by worldbeater — APPROVED** [review]

**Comment by worldbeater** [comment]
Going to bump the version number in a separate PR and 🚢 a new version after merging this.

**Comment by worldbeater** [comment]
This is now deployed to NuGet as 1.3.1 https://github.com/AvaloniaCommunity/Live.Avalonia/releases/tag/1.3.1

**Pull request merged**